psycop/common/cohort_definition.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+
+import polars as pl
+from pydantic import BaseModel
+
+
+@dataclass
+class StepDelta:
+    step_name: str
+    n_before: int
+    n_after: int
```

Comment:
```suggestion
    n_after: int
    step_index: int
```

Currently, FilteredCohort has a list of StepDeltas. Whether this is a queue or a stack will determine the ordering. I'd love to have this in each StepDelta as well, to ensure that reordering the list doesn't screw up the ordering :-)